### DISCUSSION TITLE
how to set the node same width at the same level?

### DISCUSSION THREAD

**Discussion by bigbear1024** [markmap/markmap#241] [General]
I want my mindmap look neatness, how to let same level's node with same width? edit css syntax?
![1712739482047](https://github.com/markmap/markmap/assets/50311460/dfe0df65-efc5-4b02-82ba-fd6a1840bdc2)

**Comment by gera2ld** [comment]
It's not supported, maybe we can add an option for it.
As a workaround you may set the width of each element to a fixed value in css.
